js/components/utils.js

```diff
@@ -84,23 +84,25 @@ var Utils = {
   * @return {String} The time as a string in the HH:MM format
```

Comment:
Should it be documented that nothing is returned if the timeInSeconds is not finite?